x-pack/plugins/apm/public/components/shared/charts/timeline/marker/error_marker.tsx

```diff
@@ -46,7 +46,7 @@ const Button = euiStyled(Legend)`
 // We chose 240 characters because it fits most error messages and it's still easily readable on a screen.
 function truncateMessage(errorMessage?: string) {
   const maxLength = 240;
-  if (typeof errorMessage === 'string' && errorMessage.length > maxLength) {
+  if (errorMessage && errorMessage.length > maxLength) {
```

Comment:
If you initialize the `errorMessage = ''` you can skip the first check.

```suggestion
  if (errorMessage.length > maxLength) {
```

Reply:
This was a bad merge. Undoing.